app/src/main/java/com/zeapo/pwdstore/crypto/PgpActivity.kt

```diff
@@ -100,6 +102,12 @@ class PgpActivity : AppCompatActivity(), OpenPgpServiceConnection.OnBound {
     }
     private var mServiceConnection: OpenPgpServiceConnection? = null
 
+    private val receiver = object : BroadcastReceiver() {
+        override fun onReceive(p0: Context?, p1: Intent?) {
```

Comment:
Please make the parameter names match the overriden method.

Reply:
Done